### PR TITLE
yt-dlp: Add version 2021.09.25

### LINKED ISSUE #2606: [Request] yt-dlp
Repo is [here](https://github.com/yt-dlp/yt-dlp)

Works like youtube-dl in some ways [here are the differences compared to default behavior](https://github.com/yt-dlp/yt-dlp#differences-in-default-behavior) but it's under active developement and add some [new features](https://github.com/yt-dlp/yt-dlp#new-features)

### PR DESCRIPTION
**Pull request by KaranKad**
Resolves #2606 
Closes #2249

### CODE REVIEW

File: bucket/yt-dlp.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    },
+    "bin": "yt-dlp.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/yt-dlp/yt-dlp/releases/download/$version/yt-dlp.exe"
+            },
+            "32bit": {
+                "url": "https://github.com/yt-dlp/yt-dlp/releases/download/$version/yt-dlp_x86.exe"
```
Comment (pratikpc) [inline bucket/yt-dlp.json:27]: Add #/yt-dlp.exe here as well

**Comment by KaranKad** [comment]
@issaclin32 is it good to go?

**Review by dwettstein — APPROVED** [review]
LGTM

**Pull request merged**